gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-default-page/api-documentation-v4-default-page.component.html

```diff
@@ -35,7 +35,7 @@
     } @else if (!isLoading) {
       <api-documentation-empty-state
         emptyPageTitle="No homepage set"
-        emptyPageMessage="You have noxt set up a homepage yet."
+        emptyPageMessage="You have haven't set up a homepage yet."
```

Comment:
```suggestion
        emptyPageMessage="You have not set up a homepage yet."
```

Reply:
@ytvnr I did it according to the design.
<img width="989" alt="Screenshot 2024-07-23 at 13 44 26" src="https://github.com/user-attachments/assets/c5e065e0-7965-46ea-900c-906c89219f57">

https://www.figma.com/design/fMXSfpGCQGVq39MgehrD6f/Documentation?node-id=665-5798&t=fB0N6Z2wIAeGHBxd-0

Looks like it will be the same for the "Terms and Conditions" in the future.

Reply:
To verify with Audrey + Anthony, but I would vote for `have not` rather than `haven't`.

Reply:
that was my feeling with a comment you made on "cannot" some times ago

Reply:
<img width="916" alt="Screenshot 2024-07-23 at 16 17 57" src="https://github.com/user-attachments/assets/74defffa-d99f-4b88-b21a-525354942722">
According to Audrey we can to use "haven't"

Reply:
@ytvnr